src/phoenix/config.py

```diff
@@ -201,6 +227,46 @@ def get_env_refresh_token_expiry() -> timedelta:
         )
 
 
+def get_env_smtp_enabled() -> bool:
+    return bool(get_boolean_env_var(ENV_PHOENIX_SMTP_ENABLED))
+
+
+def get_env_smtp_username() -> Optional[str]:
+    return os.getenv(ENV_PHOENIX_SMTP_USERNAME)
+
+
+def get_env_smtp_password() -> Optional[str]:
+    return os.getenv(ENV_PHOENIX_SMTP_PASSWORD)
+
+
+def get_env_smtp_mail_from() -> Optional[str]:
+    return os.getenv(ENV_PHOENIX_SMTP_MAIL_FROM)
+
+
+def get_env_smtp_hostname() -> str:
+    if (v := os.getenv(ENV_PHOENIX_SMTP_HOSTNAME)) is None:
+        return "127.0.0.1"
```

Comment:
Confused about this default. How does this work?

Reply:
a side car maybe? anyway, this is moot if we use `SMTP_CONNECTION_URL`